Build failure with -fno-common

GCC 10 defaults to -fno-common. This causes the following build error (gcc 9.2 + CFLAGS="-fno-common"):

```
===> LD release/quake2
cc build/client/src/backends/generic/misc.o build/client/src/client/cl_cin.o build/client/src/client/cl_console.o build/client/src/client/cl_download.o build/client/src/client/cl_effects.o build/client/src/client/cl_entities.o build/client/src/client/cl_input.o build/client/src/client/cl_inventory.o build/client/src/client/cl_keyboard.o build/client/src/client/cl_lights.o build/client/src/client/cl_main.o build/client/src/client/cl_network.o build/client/src/client/cl_parse.o build/client/src/client/cl_particles.o build/client/src/client/cl_prediction.o build/client/src/client/cl_screen.o build/client/src/client/cl_tempentities.o build/client/src/client/cl_view.o build/client/src/client/curl/download.o build/client/src/client/curl/qcurl.o build/client/src/client/input/sdl.o build/client/src/client/menu/menu.o build/client/src/client/menu/qmenu.o build/client/src/client/menu/videomenu.o build/client/src/client/sound/sdl.o build/client/src/client/sound/ogg.o build/client/src/client/sound/openal.o build/client/src/client/sound/qal.o build/client/src/client/sound/sound.o build/client/src/client/sound/wave.o build/client/src/client/vid/glimp_sdl.o build/client/src/client/vid/vid.o build/client/src/common/argproc.o build/client/src/common/clientserver.o build/client/src/common/collision.o build/client/src/common/crc.o build/client/src/common/cmdparser.o build/client/src/common/cvar.o build/client/src/common/filesystem.o build/client/src/common/glob.o build/client/src/common/md4.o build/client/src/common/movemsg.o build/client/src/common/frame.o build/client/src/common/netchan.o build/client/src/common/pmove.o build/client/src/common/szone.o build/client/src/common/zone.o build/client/src/common/shared/flash.o build/client/src/common/shared/rand.o build/client/src/common/shared/shared.o build/client/src/common/unzip/ioapi.o build/client/src/common/unzip/miniz.o build/client/src/common/unzip/unzip.o build/client/src/server/sv_cmd.o build/client/src/server/sv_conless.o build/client/src/server/sv_entities.o build/client/src/server/sv_game.o build/client/src/server/sv_init.o build/client/src/server/sv_main.o build/client/src/server/sv_save.o build/client/src/server/sv_send.o build/client/src/server/sv_user.o build/client/src/server/sv_world.o build/client/src/backends/unix/main.o build/client/src/backends/unix/network.o build/client/src/backends/unix/signalhandler.o build/client/src/backends/unix/system.o build/client/src/backends/unix/shared/hunk.o -Wl,-O1 -Wl,--as-needed -lm -ldl -rdynamic -fvisibility=hidden -Wl,--no-undefined -lm -ldl -rdynamic -fvisibility=hidden -Wl,--no-undefined -Wl,-z,origin,-rpath='$ORIGIN/lib' -Wl,-z,origin,-rpath='/usr/share/games/quake2/lib' -L/usr/lib64 -lSDL2 -o release/quake2
/usr/lib/gcc/x86_64-pc-linux-gnu/9.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: build/client/src/client/input/sdl.o:(.bss+0x8): multiple definition of `sensitivity'; build/client/src/client/cl_main.o:(.bss+0x9f3a0): first defined here
/usr/lib/gcc/x86_64-pc-linux-gnu/9.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: build/client/src/client/input/sdl.o:(.bss+0x38): multiple definition of `lookstrafe'; build/client/src/client/cl_main.o:(.bss+0x9f3a8): first defined here
/usr/lib/gcc/x86_64-pc-linux-gnu/9.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: build/client/src/client/input/sdl.o:(.bss+0x10): multiple definition of `m_yaw'; build/client/src/client/cl_main.o:(.bss+0x9f390): first defined here
/usr/lib/gcc/x86_64-pc-linux-gnu/9.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: build/client/src/client/input/sdl.o:(.bss+0x28): multiple definition of `m_pitch'; build/client/src/client/cl_main.o:(.bss+0x9f398): first defined here
/usr/lib/gcc/x86_64-pc-linux-gnu/9.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: build/client/src/client/input/sdl.o:(.bss+0x20): multiple definition of `m_side'; build/client/src/client/cl_main.o:(.bss+0x9f380): first defined here
/usr/lib/gcc/x86_64-pc-linux-gnu/9.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: build/client/src/client/input/sdl.o:(.bss+0x30): multiple definition of `m_forward'; build/client/src/client/cl_main.o:(.bss+0x9f388): first defined here
/usr/lib/gcc/x86_64-pc-linux-gnu/9.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: build/client/src/client/input/sdl.o:(.bss+0x40): multiple definition of `freelook'; build/client/src/client/cl_main.o:(.bss+0x9f438): first defined here
/usr/lib/gcc/x86_64-pc-linux-gnu/9.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: build/client/src/client/vid/vid.o:(.bss+0xf8): multiple definition of `vid_fullscreen'; build/client/src/client/input/sdl.o:(.bss+0x48): first defined here
collect2: error: ld returned 1 exit status
```

The patch below fixes it, but probably removing input-related code from `src/client/cl_main.c` is a cleaner solution? I don't know if anything depends on it.

```
diff --git a/src/client/input/sdl.c b/src/client/input/sdl.c
index 3c7356fa..6af18db3 100644
--- a/src/client/input/sdl.c
+++ b/src/client/input/sdl.c
@@ -62,15 +62,15 @@ int sys_frame_time;
 qboolean joy_altselector_pressed = false;
 
 // Console Variables
-cvar_t *vid_fullscreen;
-cvar_t *freelook;
-cvar_t *lookstrafe;
-cvar_t *m_forward;
-cvar_t *m_pitch;
-cvar_t *m_side;
+extern cvar_t *vid_fullscreen;
+extern cvar_t *freelook;
+extern cvar_t *lookstrafe;
+extern cvar_t *m_forward;
+extern cvar_t *m_pitch;
+extern cvar_t *m_side;
 cvar_t *m_up;
-cvar_t *m_yaw;
-cvar_t *sensitivity;
+extern cvar_t *m_yaw;
+extern cvar_t *sensitivity;
 
 static cvar_t *exponential_speedup;
 static cvar_t *in_grab;
```

--- a/src/client/cl_main.c
+++ b/src/client/cl_main.c
@@ -35,8 +35,6 @@ void CL_Connect_f(void);
 void CL_Rcon_f(void);
 void CL_CheckForResend(void);
 
-cvar_t *freelook;
-
 cvar_t *rcon_client_password;
 cvar_t *rcon_address;
 
@@ -57,14 +55,6 @@ cvar_t *cl_showclamp;
 
 cvar_t *cl_paused;
 cvar_t *cl_loadpaused;
-
-cvar_t *lookstrafe;
-cvar_t *sensitivity;
-
-cvar_t *m_pitch;
-cvar_t *m_yaw;
-cvar_t *m_forward;
-cvar_t *m_side;
 
 cvar_t *cl_lightlevel;
 
@@ -516,14 +506,6 @@ CL_InitLocal(void)
 	cl_anglespeedkey = Cvar_Get("cl_anglespeedkey", "1.5", 0);
 
 	cl_run = Cvar_Get("cl_run", "0", CVAR_ARCHIVE);
-	freelook = Cvar_Get("freelook", "1", CVAR_ARCHIVE);
-	lookstrafe = Cvar_Get("lookstrafe", "0", CVAR_ARCHIVE);
-	sensitivity = Cvar_Get("sensitivity", "3", CVAR_ARCHIVE);
-
-	m_pitch = Cvar_Get("m_pitch", "0.022", CVAR_ARCHIVE);
-	m_yaw = Cvar_Get("m_yaw", "0.022", 0);
-	m_forward = Cvar_Get("m_forward", "1", 0);
-	m_side = Cvar_Get("m_side", "1", 0);
 
 	cl_shownet = Cvar_Get("cl_shownet", "0", 0);
 	cl_showmiss = Cvar_Get("cl_showmiss", "0", 0);

--- a/src/client/header/client.h
+++ b/src/client/header/client.h
@@ -302,6 +302,7 @@ extern	cvar_t	*cl_timedemo;
 extern	cvar_t	*cl_vwep;
 extern	cvar_t  *horplus;
 extern	cvar_t	*cin_force43;
+extern	cvar_t	*vid_fullscreen;
 
 typedef struct
 {

--- a/src/client/input/sdl.c
+++ b/src/client/input/sdl.c
@@ -62,7 +62,6 @@ int sys_frame_time;
 qboolean joy_altselector_pressed = false;
 
 // Console Variables
-cvar_t *vid_fullscreen;
 cvar_t *freelook;
 cvar_t *lookstrafe;
 cvar_t *m_forward;
@@ -1246,7 +1245,6 @@ IN_Init(void)
 	joy_axis_triggerleft_threshold = Cvar_Get("joy_axis_triggerleft_threshold", "0.15", CVAR_ARCHIVE);
 	joy_axis_triggerright_threshold = Cvar_Get("joy_axis_triggerright_threshold", "0.15", CVAR_ARCHIVE);
 
-	vid_fullscreen = Cvar_Get("vid_fullscreen", "0", CVAR_ARCHIVE);
 	windowed_mouse = Cvar_Get("windowed_mouse", "1", CVAR_USERINFO | CVAR_ARCHIVE);
 
 	Cmd_AddCommand("+mlook", IN_MLookDown);

